src/core.c/Str.pm6

```diff
@@ -772,7 +772,7 @@ my class Str does Stringy { # declared in BOOTSTRAP
          )
         )
     }
-    multi method Numeric(Str:D: --> Numeric:D) {
+    multi method Numeric(Str:D: :$fail-or-nil --> Numeric:D) {
```

Comment:
should the `$fail-or-nil` parameter be typed Bool here and in the val sub?

Reply:
Force pushed that change.